model/error.yaml

```diff
@@ -0,0 +1,34 @@
+groups:
+  - id: error
+    type: attribute_group
+    prefix: error
+    brief: >
+      This document defines the shared attributes used to
+      report an error.
+    attributes:
+      - id: description
+        brief: 'Describes a class of error operation has ended with.'
+        type:
+          allow_custom_values: true
+          members:
+            - id: empty
+              value: ""
+              brief: 'No error. Default value.'
+            - id: other
+              value: "_OTHER"
+              brief: 'Any error reason instrumentation does not define custom value for.'
+        examples: ['timeout', 'java.net.UnknownHostException', 'server_certificate_invalid', 'Internal Server Error']
+        note: |
+          The error description SHOULD be predictable and SHOULD have low cardinality.
+          Instrumentations SHOULD document the list of errors they report.
```

Comment:
This does not sound like a description, but more like an identifier or code.

Reply:
discussed on semconv SIG and **preliminary** consensus is on `error.id`